local-test-infra/local-partner-pod.yaml

```diff
@@ -5,7 +5,9 @@ metadata:
   labels:
     app: partner
     test-network-function.com/generic: orchestrator
+    test-network-function.com/container: target
   annotations:
+    test-network-function.com/container_tests: "[\"PRIVILEGED_POD\",\"PRIVILEGED_ROLE\"]" # optional
     test-network-function.com/defaultnetworkinterface: "\"eth0\""
```

Comment:
Unrelated to this patch, but is defaultnetworkinterface optional?  I believe we should make it optional, and default to something else if not specified.

Reply:
It is optional, I didn't comment it as such though. If not set the autoconfig mechanism will attempt to find a suitable interface, but the field used is managed by OCP so won't be present in pure Kubernetes: https://github.com/test-network-function/test-network-function/blob/main/README.md#network-interfaces